src/app/(app)/components/index.tsx

```
@@ -44,19 +45,40 @@ const AndroidPDFViewer = dynamic(
+}: {
+  invoiceData: InvoiceData;
+  errorWhileGeneratingPdfIsShown: boolean;
+}) => {
   const { isAndroid } = useDeviceContext();
 
+  // Render the appropriate template based on the selected template
+  const renderTemplate = () => {
+    switch (invoiceData.template) {
+      case "stripe":
+        return <StripeInvoicePdfTemplate invoiceData={invoiceData} />;
+      case "default":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)